forms-flow-api/migrations/versions/960477800395_workflow_migration_changes.py

```diff
@@ -0,0 +1,35 @@
+"""Workflow Migration changes
+
+Revision ID: 960477800395
+Revises: 8feb43e1e408
+Create Date: 2024-10-29 11:24:12.569841
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import text
+
+# revision identifiers, used by Alembic.
+revision = '960477800395'
+down_revision = '8feb43e1e408'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('form_process_mapper', sa.Column('is_migrated', sa.Boolean(), nullable=True, comment="Is workflow migrated", server_default='false'))
```

Comment:
We will also need to set the value to TRUE for workflows (process) with only one form linked, right ?

Reply:
As of now, it's done as part of getting by process key API. After fetching from camunda & is_migrated set to True for workflows (process) with only one form. 
![image](https://github.com/user-attachments/assets/bd89c656-3306-4822-b2be-72ec518ccd1d)